sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2-beta/async_samples/sample_analyze_document_async.py

```diff
@@ -0,0 +1,173 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_analyze_document_async.py
```

Comment:
I think we should maybe add prebuilt into the sample file name

Reply:
What about general document? I didnt want to add prebuilt only to this one.

Reply:
prebuilt makes the most sense to me since we pass "prebuilt-document". do you mean you don't want to add it because the other samples don't have prebuilt? I agree that it's not ideal, but if the service team markets it as the "prebuilt document" model I think we should probably follow suit so people can connect the sample with the feature. Have you seen them use any other names for it?